Sources/Settings/Controller/PasscodeLockController.swift

```diff
@@ -156,20 +156,15 @@ class PasscodeLockController: UIViewController {
                                     for: .editingChanged)
     }
 
-    private func setNavBarAppearance() {
```

Comment:
now navigationBar changes color without this code (cause of extension) - and without lag!